src/xdocs/config_filters.xml

```diff
@@ -1799,18 +1799,19 @@ catch (Throwable th) { ... }
 &lt;/module&gt;
             </source>
             <source>
-public static final int lowerCaseConstant; // CHECKSTYLE IGNORE ConstantNameCheck FOR NEXT 3 LINES
```

Comment:
The modifier `public` has been removed to fit the "100 character per line"  limit in .java counterpart.